hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+# 4. Contucted Tx fundamental power and harmonics measurement
+The fundamental power and harmonics measurement are the basic test items for Tx performance. Usually The test will deploy a single tone signal at its maximum 
+output power state, then the CW signal will be injected into a SA by a RF cable for the measurement. The RF cable should be attached to the 50 ohm point, and the antenna and its matching circuits should be removed from the RF cable attaching point. For generating the desire CW signal, it can use the Railtest tool to control the radio board to do that. As the Railtest tool is already generated in chapter 3. It can be used for this test directly.
+## 4.1. RailTest commands for Tx CW signal
+The RailTest commands are listed below: 
+  - :>rx 0
+  - :>Setchannel 0 
+  - :>SetPower [max]  
+  - :>SetTxtone 1  
+  - :>SetTxtone 0
+  - :>setchannel 1
+  - :>SetTxtone 1  
+  - :>SetTxtone 0
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command;
+- Setchannel 0 will select a frequency points to test, it can also select another frequency point to test by changing the channel number;
+- It can use SetPower [max] to set maximum output power;
+- The "SetTxTone 1" command sets the radio to CW transmit mode, and the radio will transmit CW signal. 
+- The CW signal transmission can be disabled by "SetTXtone 0" command. Then the Setchannel command can be issued in idle radio state.
+
+## 4.2. The Signal Analyser setup and test example
+It is unlike CTune calibration, the fundamental and harmonics test requires big frequency span and big RBW for fast sweep measurement.
+- Set appropriate reference level for the test, e.g. if your maximum output power is +20dBm, you can set reference level to +23dBm which is greater than maximum power by 2-3 dB;
+- Set the frequency span to contain the fundamental and desired harmonics for the test. Usually 5 times fundamental frequency is desired. But according to the regulation, the maximum harmonic is tested at 10 times of fundament;
+- Set the RBW as big as possible, as long as the noise floor of SA should below the limit by 3 dB to 10dB.
+- Set the VBW to 3 times of RBW;
+- Set the trace mode to Max Hold;
+- Set the detector mode to Peak detector;
+- It can set the cable loss in fundamental and harmonics frequency points, this will count the cable loss in the display results;
+- It can set a marker table to display the signal strenghth at fundamental and harmonics.
+
+The following picture is a test example:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\Tx-Power-and-Harmonics.png">  
+</div>
+
+
+# 5. Conducted Rx sensitivity measurement
+Sensitivity is basic test item for Rx performance. Usually it use a Signal Generator(SG) to generate a modulated signal and inject it into radio chip. Then the radio chip can demodulate the signal into data, the Railtest tool can use the data to count the BER. Once the BER critia is achieved, then the SG power level is called sensitivity.
```

Comment:
Insert space before (SG).  Should we also mention PER or is the focus BER testing?

Reply:
The PER test is very similar with BER. It just show the test method to customer focus BER here.